test/libsolidity/syntaxTests/abiEncoder/passing_internal_function_pointer_to_abi_encodecall.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
```

Comment:
The original tests for `abi.encodeCall()` in `syntaxTests/specialFunctions/`. I'd more this one there too to keep them in one place.

This one could be `syntaxTests/specialFunctions/encodeCall_fail_args_internal_function_ptr_for_uint.sol` and the other `syntaxTests/specialFunctions/encodeCall_fail_args_internal_struct_for_uint.sol`.